CODEOWNERS

```diff
@@ -175,6 +175,8 @@
 #   subsystems such as xfrm (IPsec), iptables / nftables, tc. Maintain the
 #   control plane layers that populate most eBPF maps; account for endianness
 #   and system architecture impacts on the datapath code.
+# - @cilium/sig-foundations:
+#   Maintain the core libraries and frameworks.
```

Comment:
In general the idea behind these descriptions is to help provide some guidelines about special "gotchas" or context-specific knowledge that a reviewer can use in order to provide useful feedback on PRs that modify these code areas.

I wonder if we can expand on this a bit, for instance encouraging careful consideration for common APIs with ideally a single solution that works for all use cases rather than proliferating many APIs for similar use cases.

I also note that the @cilium/sig-agent already covers `Internal architecture, core data structures and daemon startup.`, so an additional question would be, do we need another separate SIG for this or could we fold these files into @cilium/sig-agent ? Is there something materially different between the review of the core agent logic vs. the files that this PR is proposing to be reviewed by a new group?

Reply:
Reworded the description, PTAL. The question regarding adding another SIG we resolved during the community meeting on May 3rd.